packages/config/src/layer2s/loopring.ts

```diff
@@ -51,7 +51,7 @@ export const loopring: Layer2 = {
 
     links: {
       websites: ['https://loopring.org'],
-      apps: ['https://exchange.loopring.io/'],
+      apps: ['https://loopring.io/#/'],
```

Comment:
```suggestion
      apps: ['https://loopring.io/#/trade'],
```
so it's not the same as the `website`?

Reply:
They are a bit different, but this works as well